app/views/news/2020-04-28-April-2020-monthly-update.html.md

```diff
@@ -0,0 +1,64 @@
+---
+summary: In March, we paid for 123.5 hours of developer work, retired the Bundler repository and GitHub organization, and completed many fixes and improvements for the RubyGems.org website.
```

Comment:
Once we have the other sections written, update here with 1-2 sentences that include the total hours and maybe 1-2 things that we did.